data/releasenotes.json

```
@@ -1,4 +1,25 @@
 [
+	{
+		"version": "v2025-12-19",
+		"date": "2025-12-19",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Add VACUUM (CHECK_INTEGRITY) to verify database integrity.",
+			"<b>Performance:</b> table scans on partitioned tables are now up to 1000 times faster.",
+			"<b>Performance:</b> improve performance of complex queries on system tables.",
+			"<b>PostgreSQL Compatibility:</b> Support ALTER SEQUENCE operations: RENAME and change OWNER TO.",
+			"<b>Maintenance:</b> create sequence if defined for column in alter table and set correct default value.",
+			"<b>Maintenance:</b> Fix a race condition where we loaded data with `COPY` to tables without writing their index entries.",
+			"<b>Maintenance:</b> Fix incorrectly empty results for index scans in partitioned relations when the partition key is not bound.",
+			"<b>Maintenance:</b> Fixed an issue during database shutdown that could cause some recently modified pages not to be persisted. This often manifested as missing index entries after restart.",
```

```suggestion
			"<b>Maintenance:</b> Fix an issue during database shutdown that could cause some recently modified pages not to be persisted. This often manifested as missing index entries after restart.",
```
The other notes say "Fix"